test/models/User.spec.js

```diff
@@ -0,0 +1,94 @@
+import assert from 'power-assert';
+import mockgoose from 'mockgoose';
+import Promise from 'bluebird';
+import UserModel from 'models/User';
+
+describe('/models/User', ()=> {
+  afterEach(()=> {
+    mockgoose.reset();
+  });
+
+  it('同一のメールアドレスは登録できない', (done)=> {
+    const accounts = [
+      { mail: 'dragonball@jump.com', password: 'wakuwakusitekitazo' },
+      { mail: 'dragonball@jump.com', password: 'wakuwakusitekitazo' },
+    ];
+    Promise.map(accounts, (account)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).